prowler/lib/outputs/common.py

```diff
@@ -14,6 +14,7 @@ def fill_common_finding_data(finding: dict, unix_timestamp: bool) -> dict:
         "status_extended": finding.status_extended,
         "muted": finding.muted,
         "resource_details": finding.resource_details,
+        "resource_metadata": finding.resource_metadata,
```

Comment:
I'd leave this commented until we have all the checks completed, not to generate inconsistent outputs, some with the `resource_metadata` and some without.